pkgs/development/libraries/SDL_gpu/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, SDL2, libGLU }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
This is already implied because you set `pname` and `version`, so you can entirely omit the `name` declaration.